lib/services/server/informationReporting.js

```diff
@@ -150,9 +150,11 @@ function processStream(deviceId, objectType, objectId, resourceId, handler, obse
 function observe(deviceId, objectType, objectId, resourceId, handler, callback) {
     function createObserveRequest(obj, callback) {
         var request = {
-            host: obj.address,
```

Comment:
Please, add an entry to CHANGES_NEXT_RELEASE file about the issue this PR is fixing, e.g:

```
-Fix: <short description of the bug/issue you are fixing> (#numer_of_issue)
```

Thanks!

Reply:
Fixed in 4c663520c57ebafbcdd8ca463e586b8c28368895